contract/src/auth/api.rs

```diff
@@ -0,0 +1,23 @@
+use near_sdk::{AccountId, near, require};
+use sweat_booster_model::api::AuthApi;
+use crate::{Contract, ContractExt};
+
+#[near]
+impl AuthApi for Contract {
+    fn add_oracle(&mut self, account_id: AccountId) {
+        self.assert_oracle();
+
+        require!(self.oracles.insert(&account_id), "Oracle already registered");
+    }
+
+    fn remove_oracle(&mut self, account_id: AccountId) {
+        self.assert_oracle();
+        
+        require!(self.oracles.len() > 1, "There must be at least one oracle");
```

Comment:
[nit]
A bit clearer error message.

```suggestion
        require!(self.oracles.len() > 1, "There must be at least one oracle left after removing and oracle.");
```